packages/utils/wallet/getNativeSigner.ts

```diff
@@ -1,16 +1,17 @@
-import { SigningStargateClient } from "@cosmjs/stargate";
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate";
 
 import { getNativeWallet } from "./getNativeWallet";
 
 import { mustGetCosmosNetwork } from "@/networks";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@n0izn0iz I want to use this type, should I put this in another file, so I can use them in both places? (here and slices)

Reply:
put StoreWallet in utils/types IMO